src/components/Settings/preferences/accordions/TimeZoneAccordion/TimeZoneAccordion.tsx

```diff
@@ -57,7 +58,7 @@ export const TimeZoneAccordion: React.FC<TimeZoneAccordionProps> = ({
         enqueueSnackbar(t('Saved successfully.'), {
           variant: 'success',
         });
-        handleAccordionChange(label);
+        handleAccordionChange(null);
```

Comment:
Is closing the accordion the desired behaviour atfer saving? I might be wrong, but users might like to have it left open.